scrapinghub/client.py

```diff
@@ -247,16 +247,16 @@ def start(self, **params):
         return self._project.jobq.start(self, **params)
 
     def update(self, **params):
-        return self._project.jobq.update(self, **params)
+        return next(self._project.jobq.update(self, **params))['prevstate']
 
     def cancel(self):
         self._project.jobq.request_cancel(self)
 
     def finish(self, **params):
-        return self._project.jobq.finish(self, **params)
+        return next(self._project.jobq.finish(self, **params))['prevstate']
```

Comment:
Moved the changes into a separate PR for simplicity https://github.com/scrapinghub/python-scrapinghub/pull/46.